automatic/ruby/README.md

```diff
@@ -0,0 +1,10 @@
+# <img src="" width="48" height="48"/> [](https://chocolatey.org/packages/ruby)
+
+Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
+
+This package provides a self-contained [Windows-based installer](https://rubyinstaller.org) that includes the Ruby language, an execution environment, important documentation, and more.
+
+## Package Parameters
+
+- `InstallDir` - Ruby installation directory, by default `c:\tools\RubyXY` where XY are major and minor version parts.
```

Comment:
you should add `/` to the parameter keys.
There have been several times were users have reported they couldn't use packages with parameters,
It usually turned out that they was missing the `/` because the package parameters didn't mention it.